src/pages/ProgramAdminPage/ProgramCoverForm.tsx

```diff
@@ -104,18 +125,20 @@ const ProgramCoverForm: React.VFC<{
       return error
     }
 
-    const uploadCoverUrl = getImageSizedUrl(
-      isUseOriginSizeCoverImage,
-      coverImage
-        ? `https://${process.env.REACT_APP_S3_BUCKET}/program_covers/${appId}/${programId}/${defaultCoverId}`
-        : coverDefaultUrl,
-    )
-    const uploadMobileUrl = getImageSizedUrl(
-      isUseOriginSizeCoverMobileImage,
-      coverMobileImage
-        ? `https://${process.env.REACT_APP_S3_BUCKET}/program_covers/${appId}/${programId}/${mobileCoverId}`
-        : coverMobileUrl,
-    )
+    const defaultUploadCoverUrl = `https://${process.env.REACT_APP_S3_BUCKET}/program_covers/${appId}/${programId}/${defaultCoverId}`
+    const uploadCoverUrl = !coverMedia
+      ? coverDefaultUrl
+      : validateFile(coverMedia.name) === 'image'
+      ? getImageSizedUrl(isUseOriginSizeCoverImage, coverMedia ? defaultUploadCoverUrl : coverDefaultUrl)
+      : defaultUploadCoverUrl
+
+    const defaultUploadMobileUrl = `https://${process.env.REACT_APP_S3_BUCKET}/program_covers/${appId}/${programId}/${mobileCoverId}`
+    const uploadMobileUrl = !coverMobileMedia
+      ? coverMobileUrl
+      : validateFile(coverMobileMedia.name) === 'image'
```

Comment:
Using `coverMedia.type`.
Please try `console.log(coverMedia)` to see data  when uploading the file.

And whether `coverMobileUrl` should upload the video format needs to be confirmed.